.github/workflows/batch_build_release.yml

```diff
@@ -0,0 +1,22 @@
+name: Batch Build Release
+
+on:
+  push:
+    branches:
+      - release/**
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v1
```

Comment:
Why do we need this step, can you add a comment?

Reply:
sorry that's redundant. I've removed it 😅